src/js/controller/captions.js

```diff
@@ -132,18 +132,21 @@ define([
         }
 
         /** Listen to playlist item updates. **/
-        function _itemHandler(model, item) {
+        function _itemHandler(item) {
```

Comment:
This is still a change event callback so doesn't model need to be the first parameter? Now this method is setting `_item` to the model.